pygem/khandler.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Author:             Stefano Chiapedi
+Date:               February 28, 2017
+
+Description:
+    Derived module from filehandler.py to handle LS-DYNA keyword (.k) files.
```

Comment:
We do not follow this type of documentation. Please remove all the lines above and leave only "Derived module...".
The contribution history is incorporated in each file so every contribution is easily trackable.